docs/starting-from-scratch.md

```diff
@@ -328,20 +328,22 @@ touch home.app.js root.component.js
 Your file tree should look like this:
 
 ```bash
-|--single-spa-simple-example
-    |--node_modules
-    |--src
-        |--root-application
-            |--single-spa-config.js
-        |--home
-            |--home.app.js
-            |--root.component.js
-  |--babelrc
-  |--index.html
-  |--package.json
-  |--webpack.config.js
-  |--yarn-error.log
-  |--yarn.lock
+single-spa-simple-example
+
+├── node_modules
+├── package.json
+├── .gitignore
+└── src
+    ├── home
+        ├── home.app.js
+        ├── root.component.js
+├── .babelrc
+├── index.html
+├── single-spa.config.js
```

Comment:
If I followed this tutorial correctly, wasn't this file `index.js`

We could still have the file and have the `index.js` file call exported methods from the `single-spa-config.js` file if you want the file tree to look like this.